tox.ini

```diff
@@ -19,6 +19,7 @@ deps =
     pyinstaller==4.0
     pytest
     packaging==20.4
+    setuptools==59.8.0  # https://github.com/pypa/setuptools/issues/3089
```

Comment:
Upstream seems to be looking for a reproducible example for this bug. Any chance you can work one out? I fear they may close the issue without addressing it otherwise.